Sources/SwiftFusion/Inference/BetweenFactorAlternative.swift

```diff
@@ -30,9 +30,7 @@ public struct BetweenFactorAlternative: LinearizableFactor2 {
     let R = actualMotion.coordinate.rot.coordinate.R + (-1) * difference.rot.coordinate.R
     let t = actualMotion.t - difference.t
     
-    // TODO(fan): Discuss with Marc why this would lead to failing CI
```

Comment:
So is there an explanation for this? Just curious if it's me missing something or a compiler issue...

Reply:
It looks like a compiler issue, maybe a bug in AutoDiff's handling of subscripts or switch statements. It's good that you caught it! I have filed a bug so that we can fix the underlying issue eventually: https://bugs.swift.org/browse/TF-1300.